### PR TITLE
W-13100902-rtf-limitations-page-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by dmerlob — DISMISSED** [review]

File: modules/ROOT/pages/limitations-self.adoc [hunk 1]
```diff
@@ -1,21 +1,111 @@
...
+| Limitation | Description 
+| Node types | VM-based nodes are required. For example, Fargate is not supported. 
+| Replicas per application | The maximum number of replicas per application is 8. 
+| Associated environments per Runtime Fabric | You can associate a Runtime Fabric with up to 100 environments in any Business Group. For example, if you associate a development and a production environment with Org A and a dev environment with Org B, that is three environment associations. 
+| Business groups | You can create up to 50 Runtime Fabrics per org in a Business Group. Any sub org can contain up to 50 Runtime Fabrics, in addition to any shared by another sub org. For example, if you have parent Org A and its child Org B, you can have 50 Runtime Fabrics in Org A and 50 in Org B. You can also share all 50 Runtime Fabrics from Org A with Org B, and as a result, you will see 100 Runtime Fabrics in total in the list view of Org B.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
| Business groups | You can create up to 50 Runtime Fabrics per org in a Business Group. Any sub org can contain up to 50 Runtime Fabrics, in addition to any shared with another sub org. For example, for a parent Org A and its child Org B, you can have 50 Runtime Fabrics in Org A and 50 in Org B. You can also share all 50 Runtime Fabrics from Org A with Org B, and as a result, you will see 100 Runtime Fabrics in total in the list view of Org B. 
```
I'm not super certain about the use of abbreviations here - are `org` and `sub org` fine in this context? While `Org A` or better yet `OrgA` can be considered a name of the organisational unit the former two fall out of line, sort of.

File: modules/ROOT/pages/limitations-self.adoc [hunk 2]
```diff
@@ -1,21 +1,111 @@
...
-| Business groups | You can create up to 50 Runtime Fabrics per org in a Business Group. Any sub org can contain up to 50 Runtime Fabrics, in addition to any shared by another sub org. For example, if you have parent Org A and its child Org B, you can have 50 Runtime Fabrics in Org A and 50 in Org B. You can also share all 50 Runtime Fabrics from Org A with Org B, and as a result, you will see 100 Runtime Fabrics in total in the list view of Org B. | Yes
+| Limitation | Description 
+| Node types | VM-based nodes are required. For example, Fargate is not supported. 
+| Replicas per application | The maximum number of replicas per application is 8. 
+| Associated environments per Runtime Fabric | You can associate a Runtime Fabric with up to 100 environments in any Business Group. For example, if you associate a development and a production environment with Org A and a dev environment with Org B, that is three environment associations.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
| Associated environments per Runtime Fabric | You can associate a Runtime Fabric with up to 100 environments in any Business Group. For example, if you associate a development and a production environments with Org A and another development environment with Org B, that is three environment associations. 
```
or is the second development environment the same as the first?

File: modules/ROOT/pages/limitations-self.adoc [hunk 3]
```diff
@@ -1,21 +1,111 @@
...
\ No newline at end of file
+include::partial$include-antivirus-statement.adoc[]
+
+== Requirements for Runtime Fabric
+
+The following descriptions provide you with the general requirements for running Anypoint Runtime Fabric.
+
+=== Kubernetes Support
+
+Runtime Fabric requires a Kubernetes cluster that is provisioned and operational. Verify if your environment is correctly configured using the `rtfctl` command-line utility. See xref:install-self-managed.adoc[Install Runtime Fabric].
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
Runtime Fabric requires a Kubernetes cluster that is provisioned and operational. Verify whether your environment is correctly configured using the `rtfctl` command-line utility. See xref:install-self-managed.adoc[Install Runtime Fabric].
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 4]
```diff
@@ -1,21 +1,111 @@
...
+
+See the xref:release-notes::runtime-fabric/runtime-fabric-release-notes-2.x.x.adoc[release notes] for your major or minor version of Runtime Fabric for a list of supported Kubernetes versions.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
See the xref:release-notes::runtime-fabric/runtime-fabric-release-notes-2.x.x.adoc[release notes] for the list of supported Kubernetes versions for your Runtime Fabric.
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 5]
```diff
@@ -1,21 +1,111 @@
...
+
+=== Supported Architectures
+
+Anypoint Runtime Fabric requires worker nodes that use the x86/x64 architecture. ARM-based architectures are not supported.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
Anypoint Runtime Fabric requires worker nodes with the x86-64 architecture. ARM-based nodes are not supported.
```
unless we need to make the point for 32-bit architecture?

File: modules/ROOT/pages/limitations-self.adoc [hunk 6]
```diff
@@ -1,21 +1,111 @@
...
+
+=== Operating Systems
+
+Anypoint Runtime Fabric supports any Linux-based operating system supported by Amazon EKS, AKS, GKE, or RedHat OpenShift.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
Anypoint Runtime Fabric works on any Linux distribution supported by Amazon EKS, AKS, GKE, or RedHat OpenShift.
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 7]
```diff
@@ -1,21 +1,111 @@
...
+
+=== Nodes and Resources
+
+In general, you should follow the best practices provided by your Kubernetes vendor to ensure availability and simplify the administration of your infrastructure.
+
+Follow best practices by installing Runtime Fabric in an environment with a minimum of two nodes each having the following resources:
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
To ensure optimal performance, Runtime Fabric installation requires at least two nodes each equipped with:
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 8]
```diff
@@ -1,21 +1,111 @@
...
+
+* Minimum of two CPU cores
+* At least 15 GiB of RAM
+* At least 250 GiB of available disk space
+
+Adjust the number of nodes and amount of resources allocated according to the amount and type of workload you run on each Anypoint Runtime Fabric instance.
+
+=== Runtime Fabric ClusterRole Access
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
=== Runtime Fabric Kubernetes roles
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 9]
```diff
@@ -1,21 +1,111 @@
...
+
+The following Runtime Fabric ClusterRoles create and read data stored in Kubernetes secrets:
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
The following Runtime Fabric roles manage data stored in Kubernetes secrets:
```
Also, it feels that the purpose of this section is lacking - is this an overview of roles/entities the RTF cluster is based on or is it supposed to be actionable in any other way?

File: modules/ROOT/pages/limitations-self.adoc [hunk 10]
```diff
@@ -1,21 +1,111 @@
...
+* `rtf-restricted`                                               
+* `rtf:cluster-status`                                                   
+* `rtf:install`                                      
+* `rtf:persistence-gateway-clro-read-only`                               
+* `rtf:upgrade`
+* `rtf:mule-clusterip-service`
+
+=== Anypoint Platform Roles and Permissions
+
+To successfully use Anypoint Runtime Fabric, your Anypoint Platform account must have the following permissions enabled:
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
To successfully use Anypoint Runtime Fabric, your Anypoint Platform account must have the following permissions:
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 11]
```diff
@@ -1,21 +1,111 @@
...
+
+* To manage permissions for Anypoint Platform users, you must have the ability to use xref:access-management::index.adoc[Anypoint Access Management].
+* To deploy and manage applications ensure you have:
+** The ability to use Anypoint Runtime Manager
+** The Exchange Contributors permission enabled for your Anypoint Platform account
+** The Read Applications and Read Runtime Fabric permissions, which are needed to view all applications on the Applications page in Runtime Manager
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
** The read permissions for Applications and Runtime Fabric instances, which are needed to view applications on the Applications page in the Runtime Manager
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 12]
```diff
@@ -1,21 +1,111 @@
...
+* To use Runtime Fabric, you must have the Organization Administrators permission or the Manage Runtime Fabrics permission for the corresponding environments.
+* To delete Runtime Fabric instances, administrators need the Manage Runtime Fabrics permission at the organization level.
+
+=== Network Configuration
+
+Anypoint Runtime Fabric requires an IT administrator to configure network ports, hostnames, and certificates to function correctly. See xref:install-self-managed-network-configuration.adoc[Configuring Your Network to Support Runtime Fabric].
+
+=== Ingress Controller
+
+Runtime Fabric supports any ingress controller that is compatible with your Kubernetes environment and supports a deployment model where a separate ingress resource is created per application deployment. In general, most off-the-shelf ingress controllers support this model.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
Runtime Fabric supports any ingress controller that works well in your Kubernetes environment. It also supports a deployment model with a dedicated ingress per application. Most off-the-shelf ingress controllers support this model.
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 13]
```diff
@@ -1,21 +1,111 @@
...
+
+[IMPORTANT]
+====
+The ingress controller included with GKE provisions a separate HTTP load balancer per application by default. Before using the ingress controller provided by GKE, learn more about its behavior, exploring workarounds, or using another ingress controller if this behavior is undesirable. See the following link:https://help.mulesoft.com/s/article/Default-Ingress-Controller-Behavior-with-Runtime-Fabric-on-GKE[KB article] for more details.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
The default behavior of the ingress controller included with GKE is to provide a separate HTTP load balancer for each application. If you find this behavior undesirable, explore possible solutions or consider using another ingress controller. See the following link:https://help.mulesoft.com/s/article/Default-Ingress-Controller-Behavior-with-Runtime-Fabric-on-GKE[KB article] for more details.
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 14]
```diff
@@ -1,21 +1,111 @@
...
+====
+
+=== Logs
+
+For Titanium customers, Anypoint Runtime Fabric supports logging using Anypoint Monitoring. See xref:manage-monitor-applications.adoc#logs[Logs] for more information.
+
+Applications deployed on Runtime Fabric direct logs to `stdout`. The container runtime collects these logs and writes them to a file. The storage location of this file depends on your container runtime and configuration. Refer to the documentation for your Kubernetes environment for details.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
It is common practice for applications deployed on Runtime Fabric to log directly to `stdout`. The container runtime stores the console log in a file on disk. The storage location of this file depends on your container runtime and configuration. Refer to the documentation for your Kubernetes environment for details.
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 15]
```diff
@@ -1,21 +1,111 @@
...
+  
+=== External Log Forwarding
+
+Anypoint Runtime Fabric does not include external log forwarding. You are responsible for installing, configuring, and managing an external log forwarder. You can use any external log forwarding agent that is compatible with your Kubernetes environment running on Amazon EKS, AKS, or GKE. Common log forwarding agents include:
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
Anypoint Runtime Fabric does not forward logs to an external system. You are responsible for installing and managing this configuration. Any external log forwarding agent compatible with your Kubernetes environment running on Amazon EKS, AKS, or GKE can be used. Commonly used log forwarding agents include:
```

File: modules/ROOT/pages/limitations-self.adoc [hunk 16]
```diff
@@ -1,21 +1,111 @@
...
+
+* Splunk Connect for Kubernetes
+* Fluentbit
+
+Runtime Fabric also supports Log4j appenders.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
Runtime Fabric supports Log4j appenders. 
```
removing `also` as previous section was not about something supported by RTF.

File: modules/ROOT/pages/limitations-self.adoc [hunk 17]
```diff
@@ -1,21 +1,111 @@
...
+* Splunk Connect for Kubernetes
+* Fluentbit
+
+Runtime Fabric also supports Log4j appenders. 
+
+=== Monitoring
+
+xref:monitoring::index.adoc[Anypoint Monitoring] provides metrics for applications and API gateways deployed to Runtime Fabric. 
+
+To collect metrics, Anypoint Monitoring sidecars run in all applications deployed to Runtime Fabric. See xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric] for details, including how to enable or disable monitoring.
```
Comment (a-palchikov) [inline modules/ROOT/pages/limitations-self.adoc]:
```suggestion
To collect metrics, Anypoint Monitoring employs sidecar containers as part of each application deployment on Runtime Fabric. See xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric] for details, including how to manage monitoring. 
```

**Review by dmerlob — APPROVED** [review]

**Pull request merged**